src/main/java/com/powsybl/openloadflow/network/MostMeshedSlackBusSelector.java

```diff
@@ -41,7 +45,7 @@ public SelectedSlackBus select(List<LfBus> buses, int limit) {
         // select non-fictitious and most meshed bus among buses with the highest nominal voltage
         List<LfBus> slackBuses = buses.stream()
             .filter(bus -> !bus.isFictitious() && bus.getNominalV() == maxNominalV)
-            .sorted(Comparator.comparingInt((LfBus bus) -> bus.getBranches().size())
+            .sorted(Comparator.comparingInt(MostMeshedSlackBusSelector::getBranchCountConnectedAtBothSides)
```

Comment:
small question it counts lines and 2 windings transfo ? Should we add 3 windings transfo ? and maybe hvdc ?

Reply:
I think that HVDC lines should be considered as generators or loads and not as branches in most cases. It is questionnable when the HVDC line is operated in AC emulation, but this operation mode is quite an exception at that time (France Spain, France Italy).